iceoryx_utils/CMakeLists.txt

```diff
@@ -185,17 +208,23 @@ endif(PERFORM_CLANG_TIDY)
 #
 ########## exporting library ##########
 #
+
+#TODO add support for multiple include directories to setup_install_directories_and_export_package
+install(
+    DIRECTORY ${ICEORYX_PLATFORM_INCLUDES}
```

Comment:
this breaks out of tree builds. please change to
DIRECTORY ${ICEORYX_PLATFORM}/include/
and please add
$<INSTALL_INTERFACE:include/${PREFIX}>
to target_include_directories(iceoryx_platform ...